docs/migrations/OMN-1431-version-to-contract-version.md

```
@@ -0,0 +1,185 @@
+contract_version: "1.0.0"
+node_version: "1.0.0"  # Optional, add if these are golden spec examples
+```
+
+---
+
+## Step 3: Source Files (11 files, 22 references)
+
+Replace `.version` → `.contract_version` on contract objects:
+
+### Validation Layer
+| File | Lines | Pattern |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)